dataproxy/cloudflare-workers/src/index.ts

```diff
@@ -3,7 +3,13 @@ import { PrismaClient } from '@prisma/client'
 const prisma = new PrismaClient({})
 
 async function getUsers() {
-  const data = await prisma.user.findMany()
+  // we use a transaction as it has caused issues with tracing
+  // `traceparent: undefined` was being passed to the engine
+  // but we ultimately want to test that queries are sent well
+  const data = await prisma.$transaction([
+    prisma.user.findFirst(),
+    prisma.user.findMany()
+  ])
```

Comment:
But transaction was not supported on dataproxy as far as I know? Is it supported now?

Reply:
Yes, but only regular transactions are, it's interactive transactions that aren't (yet).

Reply:
Right! thanks